### PR TITLE
[8.0] Fix for sending notifications for expiring proxies

### PR DESCRIPTION
**Pull request by fstagni**
This feature has been broken since release v7r1!

In the meantime I also simplified matters, and removed table `ProxyDB_ExpNotifs` 

BEGINRELEASENOTES

*Framework
FIX: send notifications for expiring proxies

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
@andresailer what's the matter with RTD?

File: src/DIRAC/FrameworkSystem/DB/ProxyDB.py [hunk 1]
```diff
@@ -1347,134 +1334,76 @@ def useToken(self, token, requesterDN, requesterGroup):
...
+                information is:
+
+                DN: %s
 
-  The proxy you uploaded to DIRAC will expire in aproximately %d days. The proxy
-  information is:
+                If you plan on keep using this credentials, please upload a newer proxy by executing:
 
-  DN:    %s
+                $ dirac-proxy-init
```
Comment (chrisburr) [inline src/DIRAC/FrameworkSystem/DB/ProxyDB.py:1387]: Shall we make this configurable so we can chage it in the LHCbDIRAC CS?

Comment (fstagni) [reply src/DIRAC/FrameworkSystem/DB/ProxyDB.py:1387]: `lhcb-proxy-init` is there for historical reasons, not technical anymore (I know, you can run it without `lb-dirac` but do we need really that?)

Comment (chaen) [reply src/DIRAC/FrameworkSystem/DB/ProxyDB.py:1387]: I think we should keep `lhcb-proxy-init` as an executable, but I wouldn't bother making an option in the CS just for that mail

**Review by chaen — APPROVED** [review]

**Pull request merged**